COMET.Web.Common/Components/DisposableComponent.razor.cs

```diff
@@ -41,6 +43,7 @@ public abstract partial class DisposableComponent : IDisposable
         public void Dispose()
         {
             this.Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
Why calling the GC? Is this necessary?

Reply:
there to avoid a code smell but this code smell is acceptable, we should not add this line. It's my fault if it has been added, will revert that